apps/netlify/frontend/src/app/edit-site-modal.js

```diff
@@ -240,38 +284,55 @@ export const EditSiteModal = ({
                 />
               </FormControl>
               <FormControl marginBottom="spacingS">
-                <Checkbox
+                <FormControl.Label marginBottom={0}>Automatic deploys</FormControl.Label>
+                <FormControl.HelpText marginTop={0} marginBottom="spacingS">Rebuild site automatically when content or assets are published or unpublished</FormControl.HelpText>
+                <Switch
+                  id={assetDeploysId}
+                  name={assetDeploysId}
+                  isChecked={isAssetDeploysOn}
+                  className={styles.assetsDeployCheckbox}
+                  onChange={(e) => setIsAssetDeploysOn(e.target.checked)}
+                >
+                  When assets are published or unpublished
+                </Switch>
+                <Switch
+                  id={deploysId}
                   name={deploysId}
                   isChecked={isDeploysOn}
-                  helpText="Rebuild site when an entry of matching content types or assets are published or unpublished"
                   onChange={(e) => setIsDeploysOn(e.target.checked)}
                 >
-                  Automatic deploys on publish events
-                </Checkbox>
+                  When content is published or unpublished
+                </Switch>
               </FormControl>
               {isDeploysOn && (
                 <div className={styles.contentTypeSelect}>
-                  <Autocomplete
-                    id={contentTypeSelectId}
-                    name={contentTypeSelectId}
-                    items={availableContentTypes}
-                    emptyListMessage="There a no content types"
-                    noMatchesMessage="Your search didn't match any content type"
-                    placeholder="Add more content types..."
-                    width="full"
-                    itemToString={(item) => item.label}
-                    renderItem={(item) => (
-                      <span
-                        key={item.value}
-                        className={item.value === ALL_CONTENT_TYPES_VALUE ? styles.allContentTypes : ''}
-                      >
-                        {item.label}
-                      </span>
+                  <FormControl marginBottom={0} isInvalid={isSelectedContentTypesInvalid}>
+                    <Autocomplete
+                      id={contentTypeSelectId}
+                      name={contentTypeSelectId}
+                      items={[{ value: ALL_CONTENT_TYPES_VALUE, label: 'Select all Content Types' }, ...filteredContentTypes]}
+                      emptyListMessage="There are no content types"
+                      noMatchesMessage="Your search didn't match any content type"
+                      placeholder="Add more content types..."
```

Comment:
Can we change this to 'All content types already selected' when `availableContentTypes.length === 0`?